setup.py

```diff
@@ -0,0 +1,11 @@
+# This setup.py is required to make type annotations available to dependent projects' mypy.
+# Its metadata is nonauthoritative.
+from distutils.core import setup
+
+setup(
+    name="servox",
+    author="Blake Watters <blake@opsani.com>",
+    version = "0.11.0",
+    package_data={"servo": ["py.typed"]},
+    packages=["servo"]
+)
```

Comment:
Servox uses Poetry, please configure this accordingly in the pyroject.toml and remove the `setup.py`: 
https://github.com/opsani/servox/blob/81cba8de5adc78a96a08eec10309d15a0efed440/pyproject.toml#L10-L12

https://blog.whtsky.me/tech/2021/dont-forget-py.typed-for-your-typed-python-package/#adding-pytyped